main.go

```diff
@@ -260,20 +273,20 @@ func sendEmail(event *corev2.Event) error {
 	var contentType string
 
 	smtpAddress := fmt.Sprintf("%s:%d", config.SmtpHost, config.SmtpPort)
-	subject, subjectErr := resolveTemplate(config.SubjectTemplate, event)
+	subject, subjectErr := resolveTemplate(config.SubjectTemplate, event, ContentPlain)
 	if subjectErr != nil {
 		return subjectErr
 	}
-	body, bodyErr := resolveTemplate(emailBodyTemplate, event)
-	if bodyErr != nil {
-		return bodyErr
-	}
 
-	if strings.Contains(body, "<html>") {
-		contentType = "text/html"
-		body = strings.Replace(body, "\n", "<br>", -1)
+	if strings.Contains(emailBodyTemplate, "<html>") {
```

Comment:
This will fail to work properly for some valid HTML documents. Consider:

```
<html lang="en">
```

Perhaps it would be better to make the content type a flag of the handler? I suspect trying to detect if the content is something that can be rendered as HTML might be too hard to do reliably.

Reply:
I am to blame for the origination of that conditional.  And, honestly, I think your approach, an explicit content-type flag, would be a better way to go.

Reply:
Yes, this is not new logic. A don't see a problem with a flag.
Though the logic could be changed to below or a regex...
``` 
if strings.Contains(emailBodyTemplate, "<html") {
```

Reply:
I'm fine with adding a content type flag in a subsequent PR. I think it would be well worth having this be an explicit, user-controlled mode.